weatherman.py

```diff
@@ -0,0 +1,177 @@
+import os
+import sys
+import argparse
+import datetime
+import glob
+import csv
+
+class DayReport:
+  def __str__(self):
+    print(self.date)
+
+  def __init__(self, date, maxtemp, meantemp, mintemp, dew, meandew, mindew, maxhumidity, meanhumidity, minhumidity, maxsealevel, meansealevel, minsealevel, maxvisibility, meanvisibility, minvisibility, maxwind, meanwind, maxgust, precipitaion, cloudcover, events, winddirdegrees):
+    self.date = date
+    self.maxtemp = maxtemp
+    self.meantemp = meantemp
+    self.mintemp = mintemp
+    self.dew = dew
+    self.meandew = meandew
+    self.mindew = mindew
+    self.maxhumidity = maxhumidity
+    self.meanhumidity = meanhumidity
+    self.minhumidity = minhumidity
+    self.maxsealevel = maxsealevel
+    self.meansealevel = meansealevel
+    self.minsealevel = minsealevel
+    self.maxvisibility = maxvisibility
+    self.meanvisibility = meanvisibility
+    self.minvisibility = minvisibility
+    self.maxwind = maxwind
+    self.meanwind = meanwind
+    self.maxgust = maxgust
+    self.precipitaion = precipitaion
+    self.cloudcover = cloudcover
+    self.events = events
+    self.winddirdegrees = winddirdegrees
+    
+  
+
+def populate_readings():
+  dayslist = []
+  dayscount = 0
+  path = sys.argv[1]
+  path += r"\*.txt"
+  files = glob.glob(path)
+  for csv_file in files:
+    with open(csv_file) as file:
+      reader = csv.DictReader(file, delimiter = ",")
+      line_count = 0
+      for row in reader:
+        if("PKT" in row):
```

Comment:
if else can be avoided if we use get function of dictionary 
```
weather_date = row.get('PKT') or row['PKST]
```
get function can also receive a default argument as a second parameter if the key does not exist.